test/jsonapi/plugs/underscore_parameters_test.exs

```diff
@@ -4,89 +4,123 @@ defmodule JSONAPI.UnderscoreParametersTest do
 
   alias JSONAPI.UnderscoreParameters
 
-  test "underscores dasherized data parameters" do
-    params = %{
-      "data" => %{
-        "attributes" => %{
-          "first-name" => "John",
-          "last-name" => "Cleese",
-          "stats" => %{
-            "age" => 45,
-            "dog-name" => "Pedro"
+  describe "call/2" do
```

Comment:
Annotation: I added this `describe` block to separate the `call` tests from the `init` test.  I recommend viewing the diff with Github's `Hide Whitespace` option to make the other changes standout.